infra/base-images/base-builder/indexer/index_build.py

```diff
@@ -89,10 +89,11 @@ class Manifest:
   is_oss_fuzz: bool = False
 
 
-def _is_elf(file: Path) -> bool:
+def _is_excludable_elf(file: Path) -> bool:
```

Comment:
hmm... technically this should stay `_is_elf`, because in the other usage of this (`exclude_build_artfacts`), we do actually want to exclude all ELF.

Reply:
I feel like it's deceptive to name this is_elf when it has an unintuitive exception, do you have another name in mind?

Reply:
to clarify -- I think this exception should live elsewhere, on line 130.

Reply:
I think we would need to include it on lines 124 and 127. Do you agree?

Reply:
probably not 124 -- that's for actually excluding any form of build artifacts (e.g. when we copy /src).

Reply:
i made these changes and made things a bit safer also.